1.10/tutorials/dcos-101/app1.md

```diff
@@ -10,30 +10,30 @@ menu_order: 3
 <table class="table" bgcolor="#FAFAFA"> <tr> <td style="border-left: thin solid; border-top: thin solid; border-bottom: thin solid;border-right: thin solid;"><b>Important:</b> Mesosphere does not support this tutorial, associated scripts, or commands, which are provided without warranty of any kind. The purpose of this tutorial is to demonstrate capabilities, and may not be suited for use in a production environment. Before using a similar solution in your environment, you must adapt, validate, and test.</td> </tr> </table>
 
 # Objective
-You now have a working persistence layer -redis- running in your cluster.
-In this section you deploy the first app connecting to redis. Note that this tutorial deliberately choose to focus on the principles and to deploy a very simple app with no further logic than connecting to redis.
+You now have a working persistence layer - [Redis](https://redislabs.com/) - running in your cluster.
```

Comment:
Confirmed via section 4d of the Redis trademark guidelines that we want this capitalization. Line 8 also has a redis reference that should be capitalized.